File: Practicas/p1/PRACTICA1/Sesion2/Visitante/src/Cliente.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   Cliente.cpp
 * Author: ruben
 * 
 * Created on 7 de marzo de 2020, 15:19
 */

#include "Cliente.h"

Cliente::Cliente() {
    this->descuento = 1;
}

float Cliente::getDescuento(){
    return this->descuento;
}

File: Practicas/p1/P1_S4/s4/src/s4/Cliente.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package s4;

/**
 *
 * @author agus
 */
public class Cliente {
    public static GestorFiltros filtros;
    private static InterfazSalpicadero interfaz;

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        InterfazSalpicadero interfaz = new InterfazSalpicadero();
        interfaz.main(new String[0]);
        
    }
    
}
